core/common/src/main/java/com/alipay/sofa/rpc/common/struct/NamedThreadFactory.java

```diff
@@ -47,26 +47,30 @@
      * 是否守护线程，true的话随主线程退出而退出，false的话则要主动退出
      */
     private final boolean              isDaemon;
+    /**
+     * 线程名第一前缀
+     */
+    private final String               firstPrefix = "SOFA-";
 
     /**
      * 构造函数，默认非守护线程
      *
-     * @param prefix 前缀，后面会自动加上-T-
+     * @param prefix2 第二前缀，前面会自动加上第一前缀，后面会自动加上-T-
      */
-    public NamedThreadFactory(String prefix) {
-        this(prefix, false);
+    public NamedThreadFactory(String prefix2) {
```

Comment:
Can you change the parameter name to `secondPrefix`? Because you have a field called `firstPrefix`, so I think `secondPrefix` is more suitable then `prefix2`